x-pack/plugins/canvas/types/embeddables.ts

```diff
@@ -28,4 +32,5 @@ export type CanvasContainerApi = PublishesViewMode &
   HasDisableTriggers &
   HasType &
   HasSerializedChildState &
+  HasRuntimeChildState &
```

Comment:
Is this change still needed?

Reply:
Nope, leftover from the canvas revert, removing